Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -0,0 +1,146 @@
+*   At least one special character
+*   It must be different than current password (if available)
+
+Password policies can be configured individually for both frontend and backend
+context. It is also possible to extend a password policy with custom validation
+requirements.
+
+.. contents:: **Table of Contents**
+   :depth: 1
+   :local:
+
+
+Configuring password policies
+=============================
+
+A password policy is defined in the TYPO3 global configuration. Each policy
+must have a unique identifier (the identifier `default` is reserved by TYPO3)
+and must at least contain one validator.
+
+The password policy identifier is used to assign the defined password policy
+to the backend and/or frontend context. By default, TYPO3 uses the
+password policy `default`:
+
+..  code-block:: php
+    :caption: config/system/settings.php | typo3conf/system/settings.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordPolicy'] = 'default';
+    $GLOBALS['TYPO3_CONF_VARS']['FE']['passwordPolicy'] = 'default';
+
+..  seealso::
+
+    :ref:`Default password policy configuration <typo3ConfVars_sys_passwordPolicies>`
+
+A custom password policy with the identifier `simple` can be configured like:
+
+..  code-block:: php
+    :caption: config/system/additional.php | typo3conf/system/additional.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['SYS']['passwordPolicies'] = [
+        'simple' => [
+            'validators' => [
+                \TYPO3\CMS\Core\PasswordPolicy\Validator\CorePasswordValidator::class => [
+                    'options' => [
+                        'minimumLength' => 6,
+                    ],
+                ],
+            ],
+        ],
+    ];
+
+Then assign the custom password policy `simple` to frontend and/or backend
+context:
+
+..  code-block:: php
+    :caption: config/system/settings.php | typo3conf/system/settings.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordPolicy'] = 'simple';
+    $GLOBALS['TYPO3_CONF_VARS']['FE']['passwordPolicy'] = 'simple';
+
+..  attention::
+    When implementing a custom password policy please refer to the
+    :ref:`secure password guidelines <security-secure-passwords>`.
+
+
+Password policy validators
+==========================
+
+TYPO3 ships with two password policy validators, which are both used in the
+default password policy.
+
+:php:`\TYPO3\CMS\Core\PasswordPolicy\Validator\CorePasswordValidator`
+---------------------------------------------------------------------
+
+This validator has the ability to ensure a complex password with a defined
+minimum length and four individual requirements.
+
+The following options are available:
+
++-----------------------------------+----------------------------+------+---------+
+| Option                            | Description                | Type | Default |
++-----------------------------------+----------------------------+------+---------+
+| :php:`minimumLength`              | Minimum length             | int  | 8       |
```

Comment:
Could we maybe auto-generate this information from the class? Also the descriptions seem kind of superflous to me so auto-generated should be sufficienz

Reply:
It is only available as class comment, so I don't see a benefit to include a comment via code snippet:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/PasswordPolicy/Validator/CorePasswordValidator.php#L23

Reply:
Ok, I assumed those are properties of the class. Should we drop the description as it really doesn't provide additional information? Also I would prefere to have this as definition list not as table as definitions list are better in terms of responsiveness, accessibility and easier to edit later on.

Reply:
Yup, thought the same, moved to a confval list.